### DISCUSSION TITLE
How to get Spatie tags count as column?

### DISCUSSION THREAD

**Discussion by bartclaeys** [filamentphp/filament#5362] [Help]
I am using Spatie Laravel Tags and created a resource to show all the available tags. Now I want to add a column with the number of times this tag has been used. The following does not work:

`TextColumn::make('tags_count')->counts('tags'),
`

with error:

`Call to undefined method Spatie\Tags\Tag::tags()
`

Now the `tags` table only holds the tags while the `taggable` table holds the the items that are being tagged. The count should be on that table, but this doesn't work either:

`TextColumn::make('taggable_count')->counts('tags.taggable'),`

**Comment by danharrin** [comment]
Strangely, there is no relationship defined on the Tag model. You'd need that to render a count. PR it to Spatie's package?
